src/setupStore.js

```diff
@@ -10,17 +12,22 @@ const composeEnhancers =
     ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
     : null || compose;
 
+const persistConfig = {
+  key: 'root',
```

Comment:
should key be `root` or `auth` (i don't know the answer... just looks odd to me).

Reply:
I wasn't 100% sure what key did, hard to find any in depth documentation, Seems to it is just a string value and possibly used if we have multiple persistent stores? Doesn't break when I change it...